workflow/cases/coupled_free_forecast_wave_test.yaml

```diff
@@ -0,0 +1,60 @@
+case:
```

Comment:
What is this case file for? Can an existing one not be used?

Reply:
This file is excessive, along with the _bk file mentioned above. I will remove them.